dashboard/app/controllers/plc/enrollment_evaluations_controller.rb

```diff
@@ -0,0 +1,20 @@
+class Plc::EnrollmentEvaluationsController < ApplicationController
+  def perform_evaluation
+    authorize! :read, Plc::Course
+    @plc_user_course_enrollment = Plc::UserCourseEnrollment.find(params[:enrollment_id])
+    @questions = @plc_user_course_enrollment.plc_course.plc_evaluation_questions.all
+    @user = @plc_user_course_enrollment.user
+    @course = @plc_user_course_enrollment.plc_course
+  end
+
+  def submit_evaluation
+    authorize! :read, Plc::Course
```

Comment:
It doesn't really matter right now - only admins can read and write courses.

Eventually, this authorization needs to be more complex, you can only submit an evaluation for a user if you are that user and you are a teacher (and maybe part of X cohort or something). For the time being, I think this is enough to keep non-admins from messing about here though.